pkg/rtc/transport.go

```diff
@@ -238,3 +255,27 @@ func (t *PCTransport) createAndSendOffer(options *webrtc.OfferOptions) error {
 	go t.onOffer(offer)
 	return nil
 }
+
+func (t *PCTransport) OnStreamedTracksChange(f func(paused map[string][]string, resumed map[string][]string) error) {
```

Comment:
noticing how these are simply forwarding things.. I do wonder if it makes sense to put them in PCTransport vs just inside participant.

don't have to change now, I think eventually we'll move some participant functionality into Subscriber and Publisher

Reply:
Yeah, I think eventually it will be good. I put it in participant because it has the `writeMessage` method which made it a good place to aggregate this.